src/LiteCore/test/LiteCore.Tests.Shared/DatabaseTest.cs

```diff
@@ -62,6 +67,11 @@ private void AssertMessage(C4ErrorDomain domain, int code, string expected)
             msg.Should().Be(expected, "because the error message should match the code");
         }
 
+        private int GetPosixExist(string name)
```

Comment:
Very strange function, if this is a code that is consistent between platforms, you can just use EEXIST directly in the test without even having this method.  The reason I recommended one was in case the error was different between platforms (also why pass in the name if you don't use it?)